storage/innobase/row/row0merge.cc

```diff
@@ -2286,6 +2303,48 @@ row_merge_read_clustered_index(
 			}
 		}
 
+		if (old_table->versioned()) {
+			if (new_table->versioned() && !drop_historical) {
+				dfield_t *end = dtuple_get_nth_field(
+					row, new_table->vers_end);
+				byte *data = static_cast<byte *>(
+					dfield_get_data(end));
+				ut_ad(data);
+				if (mach_read_from_8(data) == TRX_ID_MAX) {
+					dfield_t *start = dtuple_get_nth_field(
+						row, new_table->vers_start);
+					void *data = dfield_get_data(start);
+					ut_ad(data);
+					mach_write_to_8(data, trx->id);
+					trx->vers_update_trt= true;
```

Comment:
Do we really need to assign `trx->vers_update_trt = true` for every processed row in `ALTER TABLE`?
In InnoDB, there is a space before `=`.

Why do we need this condition? What was wrong with the existing `vers_start` column value? The row was originally created at that time.